components/BottomSheet/src/MDCBottomSheetPresentationController.m

```diff
@@ -145,11 +145,11 @@ - (void)viewWillTransitionToSize:(CGSize)size
   [super viewWillTransitionToSize:size withTransitionCoordinator:coordinator];
 
   [coordinator animateAlongsideTransition:
-       ^(id<UIViewControllerTransitionCoordinatorContext>  _Nonnull context) {
-    _sheetView.frame = [self frameOfPresentedViewInContainerView];
-    [_sheetView layoutIfNeeded];
-    [self updatePreferredSheetHeight];
-  } completion:nil];
+      ^(__unused id<UIViewControllerTransitionCoordinatorContext>  _Nonnull context) {
+        _sheetView.frame = [self frameOfPresentedViewInContainerView];
```

Comment:
These should be indented 2 spaces from the start of [coordinator anim...].
(The original indent was correct)

Reply:
We spoke offline. Since it's a block, the updated formatting is more aligned with the style guide.